bedrock/products/templates/products/vpn/includes/unsupported-language.html

```diff
@@ -0,0 +1,19 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% if not vpn_supported_locale(LANG) and ftl_has_messages('vpn-shared-unsupported-language-warning', 'vpn-shared-unsupported-language-contribute') %}
```

Comment:
Note for reviewer: you may have to remove `and ftl_has_messages('vpn-shared-unsupported-language-warning', 'vpn-shared-unsupported-language-contribute')` locally in order to see the message, since it has not yet been translated.